train.py

```diff
@@ -0,0 +1,181 @@
+import json
+import random
+import time
+import warnings
+
+import spacy
+from spacy.util import compounding, minibatch
+from spacy.gold import GoldParse
+from spacy.scorer import Scorer
+from tqdm import tqdm
+import os
+import logging
+import pandas as pd
+import copy
+
+names = pd.read_csv('top100names.csv')
+
+
+def send_output(text):
+    if __name__ == '__main__':
+        print(text)
+    else:
+        logging.info(text)
```

Comment:
I think I know why you need to do this; the logging is not appearing in console because you are not using basicConfig.

You can get rid of this function and replace all instances of `send_output` with `logging.info` (see comment at line 17 for how to set it up)